interface/patient_file/ccr_pending_approval.php

```diff
@@ -47,65 +53,68 @@
     <?php Header::setupHeader(); ?>
     <title><?php echo xlt('Pending Approval');?></title>
 </head>
-<body class="body_top">
-<div class="container">
-<h2 class="text-center"><?php echo xlt('Pending Approval');?><h2>
-<form class="text-center" method="post" name="approve" "onsubmit='return top.restoreSession()'" >
-    <div class="table-responsive">
-        <table class="table table-striped" border="0">
-            <thead class="table-primary">
-                <tr>
-                    <th scope="col"><?php echo xlt('Patient Name'); ?></th>
-                    <th scope="col"><?php echo xlt('Match Found'); ?></th>
-                    <th scope="col"><?php echo xlt('Action'); ?></th>
```

Comment:
Hey @tywrenn 

I get in-depth research, seems like the outside form tag will cause the table didn't width not filled to 100%
I did an example. When you comment out form tag the width will fill to 100%.
https://codepen.io/CliffSu/pen/GRZNENE